src/main/resources/FormSectionProfile.json

```diff
@@ -13,6 +14,7 @@
       {
         "fieldName": "Label",
         "metadataPropertyName": "label",
+        "description": "Name or main prompt text for the section", 
```

Comment:
Ditto the comment about the label on FormItemProfile

Reply:
Done